docs/Hosts/Spawn-Hosts.md

```diff
@@ -173,6 +166,30 @@ This can also be done through the CLI:
 evergreen host stop --host <host_id> --keep-off
 ```
 
+#### FAQ
+
+**Q: Can I still manually stop and start my hosts if the schedule is running?**
+A: Yes! Even though Evergreen will automatically stop and start your host on the configured schedule, you can still
+manually stop and start your host whenever you need to. If you do this, it's recommended, you also [set a temporary
+exemption](#temporary-exemptions) to ensure that you can use your host without being accidentally interrupted when the
+sleep schedule tries to automatically stop/start your host.
+
+**Q: Will the hostname change between restarts?**
+
+A: You can get a hostname that doesn't change from your unexpirable host. While it's true that EC2 hosts typically do
```

Comment:
> You can get a hostname that doesn't change from your unexpirable host.
Do they need to do anything special to request it? If yes, can we add those details? 
If not, can we rephrase it to something like "You _will_ be automatically assigned a...

Reply:
It's automatically assigned, I'll mention it.